Vagrantfile

```diff
@@ -120,20 +118,46 @@ end
 
 # Ensure plugins are installed (updated Aug 31, 2018: https://stackoverflow.com/questions/19492738/demand-a-vagrant-plugin-within-the-vagrantfile/28801317#28801317).
 def ensure_plugins(plugins)
-  logger = Vagrant::UI::Colored.new
   plugins_to_install = plugins.select { |plugin| not Vagrant.has_plugin? plugin }
   if not plugins_to_install.empty?
-    logger.warn("Installing plugins: #{plugins_to_install.join(' ')}")
+    $logger.warn("Installing plugins: #{plugins_to_install.join(' ')}")
     if system "vagrant plugin install #{plugins_to_install.join(' ')}"
       # Exit after installation, to avoid https://github.com/hashicorp/vagrant/issues/2435.
-      logger.warn("Plugins installed. Please re-run the initial command.")
+      $logger.warn("Plugins installed. Please re-run the initial command.")
     else
-      logger.error("Installation of one or more plugins has failed. sudo must be used to install plugins. Aborting.")
+      $logger.error("Installation of one or more plugins has failed. sudo must be used to install plugins. Aborting.")
```

Comment:
Err, no. We don't want people to run vagrant as root. Installation of plugins are made as a standard user.